calicolipidlibrary/lipidRules.py

```diff
@@ -431,8 +441,10 @@ def __init__(self):
     # NCE = "20,30,40"
 
     # for DI libraries
-    neg_adduct_set = ["[M-H]-", "[M+Cl]-", "[M+37Cl]-", "[M+FA-H]-", "[M+AcOH-H]-"]
-    pos_adduct_set = ["[M+Li]+", "[M+H]+", "[M+Na]+", "[M+NH4]+", "[M+K]+"]
+    #neg_adduct_set = ["[M-H]-", "[M+Cl]-", "[M+37Cl]-", "[M+FA-H]-", "[M+AcOH-H]-"]
+    #pos_adduct_set = ["[M+Li]+", "[M+H]+", "[M+Na]+", "[M+NH4]+", "[M+K]+"]
+    neg_adduct_set = ["[M-H]-","[M+FA-H]-"]
+    pos_adduct_set = ["[M+H]+", "[M+Na]+", "[M+NH4]+"]
     NCE = ""
```

Comment:
You could probably delete the commented out `neg_adduct_set` and `pos_adduct_set`, or add a comment explaining the logic behind old/new adduct sets